src/documentation/pages/components/avatar/index.vue

```diff
@@ -0,0 +1,37 @@
+<template lang="pug">
+  div
+    <TitleBar title="Avatar" description="Avatars are used to represent users, groups, and entities." :tabs="pageTabs" />
+    .chi-grid__container.-pt--3
+      .chi-tabs-panel.-active#examples
+        <Examples />
+      .chi-tabs-panel#properties
+        <Properties />
+      .chi-tabs-panel#accessibility
+        <Accessibility />
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import Examples from './examples/index.vue';
+import PageContentTabs from '../../../components/PageContentTabs.vue';
+import Properties from './_properties.vue';
+import Accessibility from './_accessibility.vue';
+import { standardComponentPageTabs } from '../../../constants/constants';
+
+Vue.config.ignoredElements = ['chi-grid', 'chi-avatar'];
```

Comment:
is chi-brand not needed here (i.e. no chi-brand error?)

Reply:
Yes, not needed as there is no chi-brand error